File: Backend/Models/UserAccount.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Backend.Models
{
    [Table("UserAccount")]
    public class UserAccount
    {
        [Key]
        [Column("ID")]
        public int ID { get; set; }
        
        [Required]
        [Column("FirstName")]
        [MaxLength(50)]
        public string FirstName { get; set; }

        [Required]
        [Column("LastName")]
        [MaxLength(50)]
        public string LastName { get; set; }

        [Required]
        [Column("City")]
        [MaxLength(40)]
        public string City { get; set; }

        [Required]
        [Column("Address")]
        [MaxLength(50)]
        public string Address { get; set; }

        [Required]
        [Column("PostalCode")]
        [Range(10000,50000)]
        public int PostalCode { get; set; }

        [Required]
        [Column("PhoneNumber")]
        [MaxLength(15)]
        public int PhoneNumber { get; set; }

        [Required]
        [Column("MailAddress")]
        [MaxLength(320)]
        public string MailAddress { get; set; }

        [Column("Orders")]
        public List<Order> Orders { get; set; }
    }
}

File: Frontend/javascript/UserAccount.js
import { WebStore } from "./WebStore.js";
export class UserAccount{
    constructor(id, firstn, lastn, city, address, postal, phone, mail){
        this.id = id;
        this.firstName = firstn;
        this.lastName = lastn;
        this.city = city;
        this.address = address;
        this.postalCode = postal;
        this.phoneNumber = phone;
        this.mailAddress = mail;
        this.orders = [];
    }
    drawTable(host){
        let tableBody = host;

        let row = document.createElement("tr");
        tableBody.appendChild(row);

        let field = document.createElement("td");
        field.innerHTML = this.firstName;
        row.appendChild(field);

        field = document.createElement("td");
        field.innerHTML = this.lastName;
        row.appendChild(field);

        field = document.createElement("td");
        field.innerHTML = this.city;
        row.appendChild(field);

        field = document.createElement("td");
        field.innerHTML = this.address;
        row.appendChild(field);

        field = document.createElement("td");
        field.innerHTML = this.postalCode;
        row.appendChild(field);

        field = document.createElement("td");
        field.innerHTML = this.phoneNumber;
        row.appendChild(field);

        field = document.createElement("td");
        field.innerHTML = this.mailAddress;
        row.appendChild(field);
    }
    addOrder(order){
        this.orders.push(order);
    }

}